package.json

```diff
@@ -1,41 +1,37 @@
 {
```

Comment:
oh, actually what happened is I added `lodash.has` with npm and npm automatically sorted the deps.  All of the versions are still the same though.